client/packages/invoices/src/utils.ts

```diff
@@ -121,11 +121,7 @@ export const getNextInboundStatus = (
 export const getNextCustomerReturnStatus = (
   currentStatus: InvoiceNodeStatus
 ): InvoiceNodeStatus | null => {
-  const currentStatusIdx = customerReturnStatuses.findIndex(
-    status => currentStatus === status
-  );
-  const nextStatus = customerReturnStatuses[currentStatusIdx + 1];
-  return nextStatus ?? null;
+  return getNextInboundStatus(currentStatus);
```

Comment:
Side quest - the next statuses couldn't be found on the customer return view and so no status change button was rendering! I think a missed thing from the inbound return -> customer return change